x-pack/plugins/synthetics/public/apps/synthetics/hooks/use_synthetics_priviliges.tsx

```diff
@@ -77,3 +102,40 @@ const Unprivileged = ({ unprivilegedIndices }: { unprivilegedIndices: string[] }
     }
   />
 );
+
+const LicenseExpired = () => {
+  const { basePath } = useSyntheticsSettingsContext();
+  return (
+    <EuiEmptyPrompt
+      data-test-subj="syntheticsUnprivileged"
+      iconType="warning"
+      iconColor="warning"
+      title={
+        <h2>
+          <FormattedMessage
+            id="xpack.synthetics.license.invalidLicenseTitle"
+            defaultMessage="Invalid License"
+          />
+        </h2>
+      }
+      body={
+        <p>
+          <FormattedMessage
+            id="xpack.synthetics.license.invalidLicenseDescription"
+            defaultMessage="The Synthetics UI is not available because your current license has expired or is no longer valid."
```

Comment:
It would be cool if we took the error message as a prop and let the user know the exact reason monitor management isn't available. Not a show stopper for this fix, but a nice enhancement in a follow-up.